examples/slate-drop-or-paste-images/index.js

```diff
@@ -46,7 +46,7 @@ class Image extends React.Component {
 class Example extends React.Component {
   plugins = [
     DropOrPasteImages({
-      insertImage: (transform, file) => {
+      insertImage: (_event, transform, file, _context) => {
```

Comment:
a little fuzzy on the _ naming convention, would recommend removing it but won't fight for it